src/serializers/persons-serializer.js

```diff
@@ -0,0 +1,83 @@
+import { Serializer as JsonApiSerializer } from 'jsonapi-serializer';
+import _ from 'lodash';
+import merge from 'merge-deep';
+import moment from 'moment';
+
+import { serializerOptions } from 'utils/jsonapi';
+import { openapi } from 'utils/load-openapi';
+import { apiBaseUrl, resourcePathLink } from 'utils/uri-builder';
+
+const personResourceProp = openapi.components.schemas.PersonResult.properties.data.properties;
+const personResourceType = personResourceProp.type.enum[0];
+const personResourceAttributes = personResourceProp.attributes.allOf;
+const personCombinedAttributes = merge(personResourceAttributes[0], personResourceAttributes[1]);
+const personResourceKeys = _.keys(personCombinedAttributes.properties);
+const personResourcePath = 'persons';
+const personResourceUrl = resourcePathLink(apiBaseUrl, personResourcePath);
+
+/**
+ * Employee status code descriptions are not stored in a db so we must manage them
+ *
+ * @param {string} statusCode employee status code returned from data source
+ * @returns {string} description for the passed in status code
+ */
+const getEmployeeStatusDescrByCode = (statusCode) => (
+  {
+    A: 'Active',
+    B: 'Leave without pay but with benefits',
+    L: 'Leave without pay and benefits',
+    F: 'Leave with full pay and benefits',
+    P: 'Leave with partial pay and benefits',
+    T: 'Terminated',
+  }[statusCode]
+);
```

Comment:
Can you move this map to contrib repo? This is also part of Banner IP even though there's no validation table.